Wikipedia/UI-V5/WMFSaveableTitleCollectionViewCell.m

```diff
@@ -32,13 +41,13 @@ - (void)prepareForReuse {
     [super prepareForReuse];
     self.title = nil;
     [self.imageView wmf_reset];
-    self.imageView.image = [UIImage imageNamed:[[self class] defaultImageName]];
+    [self configureImageViewWithPlaceholder];
 }
 
 #pragma mark - Accessors
 
-+ (NSString*)defaultImageName {
-    return @"lead-default";
+-(UIImage*)placeholderImage {
```

Comment:
Is this different per instance? If not it probably should still be a class method.